src/tests/api/BavHappyPath.test.ts

```diff
@@ -16,22 +17,27 @@ describe("Test BAV End Points", () => {
 
     beforeEach(async () => {
         //Session Request
-        sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
+       sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
     });
-    it("E2E BAV End Points Happy Path Journey", async () => {
+    it.skip("E2E BAV End Points Happy Path Journey", async () => {
         expect(sessionId).toBeTruthy();
 
-        // Make sure authSession state is as expected - BAV_SESSION_CREATED
-        await getSessionAndVerifyKey(sessionId, constants.DEV_BAV_SESSION_TABLE_NAME, "authSessionState", "BAV_SESSION_CREATED");
+      
+        // Authorization
+        const authResponse = await authorizationGet(sessionId);
+        expect(authResponse.status).toBe(200);
+        expect(authResponse.data.authorizationCode).toBeTruthy();
+        expect(authResponse.data.redirect_uri).toBeTruthy();
+        expect(authResponse.data.state).toBeTruthy();
+
+        // Make sure authSession state is as expected
+        await getSessionAndVerifyKey(sessionId, constants.DEV_BAV_SESSION_TABLE_NAME, "authSessionState", "BAV_AUTH_CODE_ISSUED");
 
         // Make sure txma event is present & valid
         const sqsMessage = await getSqsEventList("txma/", sessionId, 1);
```

Comment:
Why are we still only validating 1 TxMA message? 

The logic in place prior to this PR was hitting the session endpoint and validating **1** TxMA event. 

If you're adding an additional endpoint which results in an additional TxMA event we should be validating that we get 2 in total and both are formatted correctly.